src/com/Dryft/models/User.java

```diff
@@ -1,41 +1,20 @@
 package com.Dryft.models;
 
+import com.Dryft.utils.Hasher;
```

Comment:
This is a useless import, as far as i can see

Reply:
ohh forgot to remove that!

Reply:
@Kushagra-0801 done .Check again